apps/web/components/getting-started/steps-views/UserProfile.tsx

```diff
@@ -50,9 +50,11 @@ const UserProfile = () => {
           console.error(error);
         }
 
-        await utils.viewer.me.refetch();
-        router.push("/");
-      }
+      await utils.viewer.me.refetch();
+      const redirectUrl = localStorage.getItem("onBoardingRedirect");
+      localStorage.removeItem("onBoardingRedirect");
```

Comment:
Remove from storage so this doesnt happen if the user onboards again for whatever reason.